src/components/MultiLink/Legend.vue

```diff
@@ -123,105 +126,98 @@ export default Vue.extend({
+          // Process data for bars/histogram
+          if (this.isQuantitative(attr, type)) {
+            if (type === 'node') {
+              currentData = this.graphStructure.nodes.map((d: Node | Link) => parseFloat(d[attr]));
+            } else {
+              currentData = this.graphStructure.edges.map((d: Node | Link) => parseFloat(d[attr]));
+            }
+
+            // TODO: add histogram code
+
+            xScale = scaleLinear()
+              .domain([min(currentData), max(currentData) + 1])
+              .range([this.yAxisPadding, variableSvgWidth]);
+
+            const binGenerator = histogram()
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              .domain((xScale as any).domain()) // then the domain of the graphic
+              .thresholds(xScale.ticks(15)); // then the numbers of bins
+
+            bins = binGenerator(currentData);
+
+            if (type === 'node') {
+              store.commit.addAttributeRange({ attr, min: parseFloat(min(currentData) || '0'), max: parseFloat(max(currentData) || '0') });
+            }
+
+            yScale = scaleLinear()
+              .domain([0, max(bins, (d) => d.length) || 0])
+              .range([this.svgHeight, 0]);
+
+            variableSvg
+              .selectAll('rect')
+              .data(bins)
+              .enter()
+              .append('rect')
+              .attr('x', (d) => (xScale as ScaleLinear<number, number>)(d.x0 || 0))
+              .attr('y', (d) => (yScale as ScaleLinear<number, number>)(d.length))
+              .attr('height', (d) => this.svgHeight - (yScale as ScaleLinear<number, number>)(d.length))
+              .attr('width', (d) => (xScale as ScaleLinear<number, number>)(d.x1 || 0) - (xScale as ScaleLinear<number, number>)(d.x0 || 0))
+              .attr('fill', '#82B1FF');
           } else {
-            currentData = this.graphStructure.edges.map((d: Node | Link) => d[attr]);
-          }
-          const bins = new Map([...new Set(currentData)].map(
-            (x) => [x, currentData.filter((y) => y === x).length],
-          ));
-
-          const binLabels: string[] = [];
-          const binValues: number[] = [];
-          bins.forEach((value, label) => {
-            binLabels.push(label);
-            binValues.push(value);
-          });
+            if (type === 'node') {
+              currentData = this.graphStructure.nodes.map((d: Node | Link) => d[attr]).sort();
+            } else {
+              currentData = this.graphStructure.edges.map((d: Node | Link) => d[attr]).sort();
+            }
+
+            bins = new Map([...new Set(currentData)].map(
+              (x) => [x, currentData.filter((y) => y === x).length],
+            )) as Map<string, number>;
+
+            const binLabels: string[] = [];
+            const binValues: number[] = [];
+            bins.forEach((value, label) => {
+              binLabels.push(label);
+              binValues.push(value);
+            });
```

Comment:
Optional suggestion that increases legibility:

```suggestion
            const binLabels: string[] = bins.map((value, label) => label);
            const binValues: number[] = bins.map((value, label) => value);
```

Reply:
You can't do `Map().map()` so, instead, I did:

```
            const binLabels: string[] = Array.from(bins.keys());
            const binValues: number[] = Array.from(bins.values());
```

See 2ff9b20

Reply:
Cool. Didn't realize `Map`s even had a `.forEach()` method.